wrapper/src/main/java/software/amazon/jdbc/plugin/staledns/AuroraStaleDnsPlugin.java

```diff
@@ -55,6 +55,16 @@ public class AuroraStaleDnsPlugin extends AbstractConnectionPlugin {
       Arrays.asList("connect", "initHostProvider", "notifyNodeListChanged", "*")));
   private final PluginService pluginService;
   private final AuroraStaleDnsHelper helper;
+  private static final String POSTGRESQL_READONLY_QUERY = "SELECT rt.server_id, CASE\n"
```

Comment:
Would you consider to move these queries to AuroraStaleDnsHelper? Then getVerifiedConnection() can accept driver protocol and decide what query to run.